Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetEventChannel.tsx

```diff
@@ -0,0 +1,194 @@
+﻿//******************************************************************************************************
+//  AssetEventChannel.tsx - Gbtc
+//
+//  Copyright © 2020, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  04/19/2022 - Gabriel Santos
+//       Generated original version of source code.
+//
+//******************************************************************************************************
+
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Application, OpenXDA } from '@gpa-gemstone/application-typings';
+import { PhaseSlice, MeasurmentTypeSlice } from '../Store/Store'
+import Table from '@gpa-gemstone/react-table';
+import { useSelector } from 'react-redux';
+import { LoadingIcon, ServerErrorIcon } from '@gpa-gemstone/react-interactive';
+
+declare var homePath: string;
+
+interface IProps { Asset: OpenXDA.Types.Asset }
+
+interface ChannelDetail { //TODO: Move to Gemstone
+    ID: number,
+    MeterID: string,
+    AssetID: string,
+    MeasurementTypeID: string,
+    MeasurementCharacteristicID: string,
+    PhaseID: string,
+    Name: string,
+    Adder: number,
+    Multiplier: number,
+    SamplesPerHour: number,
+    PerUnitValue: number,
+    HarmonicGroup: number,
+    Description: string,
+    Enabled: boolean,
+    ConnectionPriority: number,
+    MeterKey: string,
+    MeterName: string,
+    AssetKey: string,
+    AssetName: string,
+    MeasurementType: string,
+    MeasurementCharacteristic: string,
+    Phase: string,
+    Mapping: string,
+    SeriesTypeID: string,
+    SeriesType: string
+}
+
+
+const AssetEventChannelWindow = (props: IProps) => {
```

Comment:
```suggestion
const AssetChannelWindow = (props: IProps) => {
```
that'll make it clear that this is all Channels includign Trending.